hudi-common/src/main/java/org/apache/hudi/common/table/HoodieTableMetaClient.java

```diff
@@ -86,6 +86,7 @@ public class HoodieTableMetaClient implements Serializable {
   public static final String BOOTSTRAP_INDEX_ROOT_FOLDER_PATH = AUXILIARYFOLDER_NAME + Path.SEPARATOR + ".bootstrap";
   public static final String HEARTBEAT_FOLDER_NAME = METAFOLDER_NAME + Path.SEPARATOR + ".heartbeat";
   public static final String METADATA_TABLE_FOLDER_PATH = METAFOLDER_NAME + Path.SEPARATOR + "metadata";
+  public static final String HASHING_METADATA_FOLER_NAME = ".hashing_metadata";
```

Comment:
Typo

Reply:
Shall we call this `.bucket_index` instead? "hashing_metadata" is too generic and doesn't really hint you at what it's all about

Reply:
Fixed the typo. `.hashing_metadata` is more consistent with the code naming (e.g., `HoodieConsistentHashingMetadata`). `.bucket_index` would be more user-friendly. How about I make it `.bucket_index/consistent_hashing_metadata/`?

Reply:
SG